webcompat/views.py

```diff
@@ -281,7 +280,7 @@ def create_issue():
             abort(400)
         # Adding parameters to the form
         form = request.form.copy()
-        extra_labels = session.pop('extra_labels', None)
```

Comment:
I haven't tested yet but what is happening with 

`/issues/new?label=browser-fenix`

Reply:
This part hasn't changed much, so the label will be saved to the current session here 
https://github.com/webcompat/webcompat.com/blob/master/webcompat/views.py#L271 

And then they're retrieved in the newly added method `get_extra_labels` (where the session  will be checked first, and then the form).